images/opensearch/config/template.apko.yaml

```diff
@@ -20,9 +20,36 @@ paths:
     gid: 65532
     permissions: 0o755
     recursive: true
+  - path: /usr/share/opensearch/config
+    type: directory
+    uid: 65532
+    gid: 65532
+    permissions: 0o755
+    recursive: true
+  - path: /usr/share/opensearch/config/jvm.options.d
+    type: directory
+    uid: 65532
+    gid: 65532
+    permissions: 0o755
+    recursive: true
+  - path: /usr/share/opensearch/data
+    type: directory
+    uid: 65532
+    gid: 65532
+    permissions: 0o755
+    recursive: true
+  - path: /usr/share/opensearch/logs
+    type: directory
+    uid: 65532
+    gid: 65532
+    permissions: 0o755
+    recursive: true
```

Comment:
I know we shouldn't need these, just adding everything for now and dial back once things are working